django-stubs/contrib/auth/__init__.pyi

```diff
@@ -26,7 +25,7 @@ def login(
     request: HttpRequest, user: Optional[AbstractBaseUser], backend: Optional[Union[Type[ModelBackend], str]] = ...
 ) -> None: ...
 def logout(request: HttpRequest) -> None: ...
-def get_user_model() -> Type[Model]: ...
+def get_user_model() -> Type[AbstractBaseUser]: ...
```

Comment:
It would be ideal to add a plugin for this function.

@yankyhoffman do you want to give this a try?

Reply:
And we also need tests for it! 👍

Reply:
@sobolevn looks like that plguin code has been there for a year and a half: https://github.com/typeddjango/django-stubs/blame/a3624dec36814bf8118d2c5f1ee3fca2e4f3b48c/mypy_django_plugin/main.py#L189

Reply:
and i can also confirm that it works properly. the following code (in a project with an installed app `my_users` containing the model `MyUser`) typechecks just fine

```py
from typing import Type

from django.contrib.auth import get_user_model
from my_users.models import MyUser

x: Type[MyUser] = get_user_model()
```

Reply:
i think this change would still be nice, along with a test and a comment on `get_user_model` explaining that the plugin takes care of it

Reply:
@3n-k1 thanks! 👍